Sources/Inferno/Shaders/Transformation/Shimmer.metal

```diff
@@ -0,0 +1,119 @@
+//
+//  Shimmer.metal
+//  Inferno
+//
+//  Created by Brent Whitman on 2024-04-10.
```

Comment:
Please copy in the same header comments as the other files; it directs folks to the repository and license.